### ISSUE TITLE
search.highlighting feature breaks scrolling of left table of contents when built offline

### ISSUE THREAD

**Issue by johnthagen** [squidfunk/mkdocs-material#3073]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

Enabling `search.highlighting` breaks the rendering of the left table of contents when the site is built offline and opened from the file system.

### Expected behaviour

A scroll bar should appear when the number of pages in the left hand table of contents is longer than can fit on the screen:

<img width="431" alt="Screenshot 2021-10-02 141357" src="https://user-images.githubusercontent.com/10340167/135727574-9af34d88-e65e-4c29-846c-80dc2ec01663.png">

### Actual behaviour

When `search.highlighting` feature is enabled and the site is built and opened from the file system, the scroll bar no longer appears and the contents run off the end of the page:

<img width="336" alt="Screenshot 2021-10-02 141506" src="https://user-images.githubusercontent.com/10340167/135727617-2637b079-772f-448f-8907-86ad9c2323ea.png">

### Steps to reproduce

1. Create a site with a large number of pages that would require the left table contents to need to wrap/scroll
2. `mkdocs build`
3. Open `index.html` directly from the file system in a browser

### Package versions

- Python: 3.9.4
- MkDocs: 1.2.2
- Material: 7.3.1

### Configuration

```yaml
site_name: My Docs
theme:
  name: material
  features:
    - search.highlight
```

### System information

- Operating system: Windows 10
- Browsers: Firefox 92.0.1, Chrome 94.0.4606.71, Edge 94.0992.38

**Comment by squidfunk** [comment]
Thanks for reporting. Are you using the localsearch plugin?

**Comment by johnthagen** [comment]
@squidfunk We are not currently using the [Offline Search](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-search/#offline-search) instructions. We were content to have the search not work offline as long as everything was readable. In our use case our docs are normally read online served via NGINX, but we also distribute it as offline files for people to refer to (for example to help bootstrap the install process that later ends up serving the docs).

To be honest, the [`localsearch` docs](https://github.com/wilhelmer/mkdocs-localsearch#installation-material-v7) seemed a bit intimidating to set up. We'd love to use it if the config was more streamlined.

**Comment by squidfunk** [comment]
Yeah, so it’s more or less expected that things break. I’ll look into it if we can make it more resilient, but it’s not a bug because it’s not intended to work.

**Comment by squidfunk** [comment]
Fixed in 88ed40101. All failing `fetch` calls will now just complete, resulting in graceful handling down the chain.

**Comment by johnthagen** [comment]
> Fixed in 88ed401.

Thanks for the quick fix!

> Regarding offline search – Material for MkDocs will soon support offline search without any further configuration

That sounds really great. I think that will really lower the barrier for people to make awesome offline docs.

**Comment by johnthagen** [comment]
@squidfunk Just wanted to report that I tried out 7.3.2 on our production docs site and this issue is in fact resolved.

Thanks!
